chat-widget/src/components/webchatcontainerstateful/common/activityConverters/convertPersistentChatHistoryMessageToActivity.ts

```diff
@@ -0,0 +1,89 @@
+import { Constants } from "../../../../common/Constants";
+import botActivity from "../activities/botActivity";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const convertStringValueToInt = (value: any) => {
+    if (typeof value !== "string" || value === "") {
+        return undefined;
+    }
+
+    let result;
+    try { 
+        result = parseInt(value);
+    } catch (e) {
+        return undefined;
+    }
+
+    return isNaN(result) ? undefined : result;
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const convertPersistentChatHistoryMessageToActivity = (message: any) => {
```

Comment:
Noticing a lot of any typing in this PR. It's a big one! Might be good to make a task to come back and define interfaces.